ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -95,52 +93,52 @@
     </F.HelperText>
   </Hds::Form::Checkbox::Field>
 
-  <form.input
-    @name='client_id'
-    @type='text'
+  <Hds::Form::TextInput::Field
```

Comment:
can we add isoptional attr here?

Reply:
@DhariniJeeva `@isOptional` added back in last commit.  Thank you for the catch